DOCS-#3792, #2410: Improve "Getting Started" section [DEPRECATED] (#3898)
modin-project/modin: docs/getting_started/examples.rst

Examples and Resources 
========================

> The earlier sections of have covered the basics of using Modin to speed up your pandas workflow. Here you can find additional resource to learn about Modin. To learn more about advanced usage for Modin, please refer to :doc:`this section </advanced_usage/index>`.

Reviewer: Seems to be some words missing here: "The earlier sections of [...] have covered". Also grammar nit in the second sentence, I believe "resource" should be made plural.

Author: Nice catch! Fixed now!

Basic Usage Examples
---------------------

The following notebooks demonstrate how Modin can be used for scalable data science: 

> - Quickstart Guide to Modin [`Source <https://github.com/modin-project/modin/tree/master/examples/quickstart.ipynb>`_]

Reviewer: Getting a "Page Not Found" here as well as in a few of the exercises in the tutorial section.

Author: This will only work after this PR is merged in.

- Using Modin with the NYC Taxi Dataset [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi.ipynb>`_] 

> - Using Modin with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_] 

Reviewer 2: ```suggestion
- Using Experimental Modin in the cloud with the NYC Taxi Dataset on a AWS cluster [`Source <https://github.com/modin-project/modin/blob/master/examples/jupyter/NYC_Taxi_cloud.ipynb>`_] 
```

> - Modin for Machine Learning with scikit-learn [`Source <https://github.com/modin-project/modin/blob/master/examples/modin-scikit-learn-example.ipynb>`_] 

Author: Renaming this as usage examples instead of labelling as basic.

- Exercise 2: Speed Improvements with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_Sources/introduction/exercise_2.ipynb>`_]
- Exercise 3: Defaulting to pandas with Modin [`Source <https://github.com/modin-project/modin/blob/master/examples/tutorial/tutorial_Sources/introduction/exercise_3.ipynb>`_]

The following tutorials covers more advanced features in Modin: 

- Exercise 4: Experimental Features in Modin (Spreadsheet, Progress Bar) [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/introduction/exercise_4.ipynb>`_]
- Exercise 5: Setting up Modin in a Cluster Environment [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/cluster/exercise_5.ipynb>`_]
- Exercise 6: Running Modin in a Cluster Environment [`Source <https://github.com/modin-project/modin/blob/721d2d2fe2a46d3b7442e1ab2c7b70ad3bff0712/examples/tutorial/tutorial_Sources/cluster/exercise_6.ipynb>`_]

> Talks & Podcasts

Reviewer 2: There are two more talks that might be added.

https://www.youtube.com/watch?v=oo_lxUjsFTM&t=1s
https://www.youtube.com/watch?v=cOM82kHRwkM&t=6568s

Reviewer 3: but we have to add a note, that they're in Russian, not in English

Author: added talk links

-----------------

- `Scaling Interactive Data Science with Modin and Ray <https://www.youtube.com/watch?v=ycSf1IbBGWk>`_ (20 minute, Ray Summit 2021) 
- `Unleash The Power Of Dataframes At Any Scale With Modin <https://www.pythonpodcast.com/modin-parallel-dataframe-episode-324/>`_  (40 minute, Python Podcast 2021) 

> - `[Russian] Distributed Data Processing and XGBoost Training and Prediction with Modin <https://www.youtube.com/watch?v=oo_lxUjsFTM&t=1s>`_ (30 minute, PyCon Russia 2021)

Reviewer 2: ```suggestion
- `[Russian] Distributed Data Processing and XGBoost Training and Prediction with Modin <https://www.youtube.com/watch?v=cOM82kHRwkM&t=6568s>`_ (30 minute, PyCon Russia 2021)
```

Author: Corrected the link here.

> - `[Russian] Efficient Data Science with Modin <https://www.youtube.com/watch?v=cOM82kHRwkM&t=6568s>`_ (1.5 hour, ISP RAS Open 2021)

Reviewer 2: ```suggestion
- `[Russian] Efficient Data Science with Modin <https://www.youtube.com/watch?v=cOM82kHRwkM&t=6568s>`_ (30 minute, ISP RAS Open 2021)
```

- `Explore Python Libraries: Make Your DataFrames Parallel With Modin by Zachary Bennett <https://www.pluralsight.com/guides/explore-python-libraries:-make-your-dataframes-parallel-with-modin>`_
- `Get faster pandas with Modin, even on your laptops by Parul Pandey <https://towardsdatascience.com/get-faster-pandas-with-modin-even-on-your-laptops-b527a2eeda74>`_
- `How to speedup pandas by changing one line of code by Shrivarsheni <https://www.machinelearningplus.com/python/modin-speedup-pandas/>`_

Here are some articles written by the international community: 

- `Python Chinese Community: 用 Modin 来提速 pandas 工作流程  <https://blog.csdn.net/BF02jgtRS00XKtCx/article/details/90709222>`_
- `Analytics India: How To Accelerate Pandas With Just One Line Of Code <https://analyticsindiamag.com/how-to-accelerate-pandas-with-just-one-line-of-code-modin/>`_
- `Разработка: Ускоряем Pandas при помощи модуля modin <https://vc.ru/dev/187095-uskoryaem-pandas-pri-pomoshchi-modulya-modin>`_

If you would like your articles to be featured here, please `submit a pull request <https://github.com/modin-project/modin/pulls>`_ to let us know!

> 

Reviewer: Nit: remove empty lines.
